### PR TITLE
close kudu scanner as the API doc suggest

### PR DESCRIPTION
**Pull request by lzeiming**

### CODE REVIEW

File: presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java [hunk 1]
```diff
@@ -208,6 +208,12 @@ private PartialRow buildPrimaryKey()
     @Override
     public void close()
     {
+        try {
+            scanner.close();
```
Comment (kokosing) [inline presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java:212]: It is a pity that scanner is not closeable.

Comment (kokosing) [inline presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java:212]: How did you find this?

File: presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java [hunk 2]
```diff
@@ -208,6 +208,12 @@ private PartialRow buildPrimaryKey()
     @Override
```
Comment (kokosing) [inline presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java:208]: Can you please change the commit message to:
```
Close kudu scanner
```

File: presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java (hunk 1 above)
Comment (lzeiming) [reply presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java:212]: Our Kudu tserver's memory is all consumed and in the heap sampled heap usage diagram indicates they are consumed by row projector code gen related to scanner. And in Kudu logs, there are a lot of scanner idle expiring.

**Pull request merged**